include/ck/wrapper/layout.hpp

```diff
@@ -43,11 +49,18 @@ struct Layout
             Number<Tuple<Ts...>::Size()>{});
     }
 
-    // Generate LowerDims in Compile-time for MergeTrasform using passed Type
-    // If element of Tuple<Ts...> is also tuple, then merge (generate sequence for merge)
-    // If tuple is element, then pass through (sequence with one element)
+    /**
+     * \brief Generate LowerDims in Compile-time for MergeTrasform using
```

Comment:
LowerDims could be lower dims or even something more descriptive
Compile-time shouldn't be capitalized
MergeTrasform -> MergeTransform or "the Merge transform"
using passed Type -> using provided type.